src/OlStyler.js

```diff
@@ -225,17 +227,17 @@ function textStyle(textsymbolizer, fature, type) {
       text: new Text({
         text,
         font: `${fontStyle} ${fontWeight} ${fontSize}px ${fontFamily}`,
-        offsetX,
-        offsetY,
+        offsetX: Number(offsetX),
+        offsetY: Number(offsetY),
         rotation,
         placement,
         textAlign: 'center',
         textBaseline: 'middle',
-        stroke: new Stroke({
+        stroke: textsymbolizer.halo ? new Stroke({
           color: halo.fillOpacity && halo.fill && halo.fill.slice(0, 1) === '#' ? hexToRGB(halo.fill, halo.fillOpacity) : halo.fill,
           // wrong position width radius equal to 2 or 4
-          width: haloRadius === 2 || haloRadius === 4 ? haloRadius - 0.00001 : haloRadius,
-        }),
+          width: (haloRadius === 2 || haloRadius === 4 ? haloRadius - 0.00001 : haloRadius) * 2,
```

Comment:
Hi @AlexanderKochIBK @allartk ,
only a question:
in my test cases I needed to add this control
`haloRadius === 2 || haloRadius === 4 ? haloRadius - 0.00001 `
because when I was using offsets and and halo radius was 2 or 4 the label placement was wrong, far away from the original position.
I wondering if is this still needed, did you tried to remove it and test if the label placement is right?

Reply:
We implemented the label placement by our own, using an independent label layer (or LabelCache), where all the labels are placed. So we do not see any differences between using that control or not.

Reply:
ok, I see, thanks!